src/vulnerabilityreports/page.tsx

```diff
@@ -44,21 +42,19 @@ export class ClusterVulnerabilityReportPage extends React.Component<{ extension:
                 renderTableHeader={[
                     {title: "Name", className: "name", sortBy: sortBy.name},
                     {title: "Image", className: "repository"},
-                    {title: "Critical", className: "critical", sortBy: sortBy.critical},
-                    {title: "High", className: "high", sortBy: sortBy.high},
-                    {title: "Medium", className: "medium", sortBy: sortBy.medium},
-                    {title: "Low", className: "low", sortBy: sortBy.low},
-                    {title: "Unknown", sortBy: sortBy.unknown},
+                    {title: "Results", className: "results", sortBy: sortBy.results},
```

Comment:
In other places counts grouped by severity are called "Summary".  I'm not sure we can make this column sortable. Would is the sort algorithm?